mesop/examples/e2e/chat_test.ts

```diff
@@ -13,7 +13,7 @@ test('Chat UI can send messages and display responses', async ({page}) => {
   await page.getByRole('button', {name: 'Send prompt'}).click();
   await expect(
     page.getByRole('button', {name: 'Processing prompt...'}),
-  ).toBeAttached({timeout: 3000});
+  ).toBeAttached({timeout: 5000});
```

Comment:
I didn't get a chance to look at the playwright output, but it's possible time out issue could be here:

```
 // Need to wait for the input state to be saved before clicking.
 await page.waitForTimeout(2000);
```

The reason is that if we don't wait long enough, "Lorem ipsum" won't get saved to state. So when we click the button, there will be no input saved, which is a noop in the chat app logic.

That line is could be particularly flaky since there is no signal that the state has been saved. It just waits the 2 seconds.